docs/development/LedStrip.md

```
@@ -144,35 +112,153 @@ The datasheet can be found here: http://www.adafruit.com/datasheets/WS2812.pdf
+First setup how the LEDs are laid out so that you can visualize it later as you configure and so the flight controller knows how many LEDs there are available.
+
+There is a step by step guide on how to use the Betaflight App to configure the Led Strip feature using the Betaflight App https://oscarliang.com/setup-led-betaflight/ which was published early 2015 by Oscar Liang which may or may not be up-to-date by the time you read this.
+
+#### Advanced LED configuration
+
+The configuration values for each LED can be displayed in the CLI using the `led` command.
+
+The `led` command with no arguments prints out the current LED configuration, which can be copied for future reference.
+
+Otherwise, `led` expects two arguments - a zero-based LED index number, a space, and then a sequence of parameters in the form:
+` index ,y:ddd:mmm:cc`, where:
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)